pkg/webui/console/components/downlink-form/downlink-form.js

```diff
@@ -140,6 +144,11 @@ DownlinkForm.propTypes = {
     push: PropTypes.func.isRequired,
     replace: PropTypes.func.isRequired,
   }).isRequired,
+  skipPayloadCrypto: PropTypes.bool,
+}
+
+DownlinkForm.defaultProps = {
+  skipPayloadCrypto: false,
 }
```

Comment:
I'm not really happy with assuming `false` by default, since this value is coming from the store. If we determine the absence of the `skip_payload_crypto` prop as indicating `false` then that should be done on the selector level and the prop should be marked as required here instead.